setup.cfg

```diff
@@ -11,10 +11,14 @@ url = https://potassco.org/
 
 [options]
 packages = find:
-zip_safe = True
+package_dir =
+    =src
```

Comment:
What does the `=` sign before src mean?

Reply:
It's some syntactic sugar. It indicates that all packages are contained under the directory. It's described [here](https://setuptools.pypa.io/en/latest/userguide/declarative_config.html#using-a-src-layout).